app/components/UI/AccountList/AccountElement/index.js

```diff
@@ -119,7 +119,7 @@ class AccountElement extends PureComponent {
 
 	render() {
 		const { disabled, updatedBalanceFromStore } = this.props;
```

Comment:
`ticker` is destructured again on line 123, I suggest adding it on this line

```suggestion
		const { disabled, updatedBalanceFromStore, ticker } = this.props;
```

Reply:
Done 👍